docs/platforms/elixir/integrations/oban/index.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Oban
+description: "Sentry supports monitoring scheduled Oban jobs and automatically capturing exceptions."
+sidebar_order: 20
+---
+
+The [Oban](https://getoban.pro) integration supports:
+
+  * Capturing errors and exceptions that happen in Oban jobs.
+  * Cron monitoring for scheduled Oban jobs.
+
+The integration is built into the Sentry SDK.
+
+## Configure
+
+You can configure the Oban integration in your Sentry configuration, under the `:integrations` key:
+
+```elixir
+# For example, in config/config.exs:
```

Comment:
```suggestion
```elixir {filename:config/config.exs}
```
You can display the filename for the codeblock like this ^

Reply:
Oh nice yeah I did this before but forgot it was a think 😉